src/test_utils.jl

```diff
@@ -173,6 +313,107 @@ function test_interface(
     @test all(length.(predictions) .== size(outputs, 1))
 end
 
+function test_interface(
+    template::Template, ::Type{PointEstimate}, ::Type{SingleOutput}, ::Type{DistributionPredictInput};
+    inputs=[MvNormal(5, m) for m in 1:5], outputs=rand(1, 5),
+)
+    predictions = test_common(template, inputs, outputs)
```

Comment:
This may fail on a real Model since `test_common` will try to call `fit` with `inputs`